### PR TITLE
Page 404

### PR DESCRIPTION
**Pull request by dokato**
Switches to 404 when bad address given (#33 ).

### CODE REVIEW

File: R/pages.R [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+#' @param message404 message to display at the 404 website
+#'
+#' @export
+#' @examples
+#' page404() # div(h1("Not found"))
+#' page404(messege = "ABC") # div(h1("ABC"))
+page404 <- function(page = NULL, message404 = NULL){
+  if (is.null(page)) {
+    if (is.null(message404)){
+      return(shiny::div(shiny::h1("Not found")))
```
Comment (filipstachura) [inline R/pages.R]: Page not found?

File: R/pages.R [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Constants
+
+#' @export
+PAGE_404_ADDRESS <- "404"
```
Comment (filipstachura) [inline R/pages.R]: Maybe `PAGE_404_ROUTE`?

File: R/pages.R [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+#' 404 page
+#'
+#' The page which appear when address of visited page is wrong.
```
Comment (filipstachura) [inline R/pages.R]: I think we need to use common nomenclature. For now, I see: address, route, path and page all used in similar context.

Comment (dokato) [reply R/pages.R]: Yeah, true, thanks for spotting it!

**Pull request merged**